.github/workflows/ci-tk.yaml

```
@@ -22,7 +22,90 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.event.number || github.sha }}
   cancel-in-progress: true
 
+env:
+  LLVM_SHA_FILE: llvm-sha.txt
+  LLVM_CACHE_NUMBER: 1  # Increase to reset cache
+  WATER_REF: 1a84aa14ee74d5015ceb41bed438de4389f59b83
```

Please add a comment here, linking to https://github.com/ftynse/water
```suggestion
  # Git ref from https://github.com/ftynse/water to checkout
  WATER_REF: 1a84aa14ee74d5015ceb41bed438de4389f59b83
```